tests/core/data/test_input_transform.py

```diff
@@ -11,196 +11,140 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from functools import partial
-from typing import Callable, Dict, Optional
+from typing import Callable
 
 import pytest
-import torch
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
 from torch.utils.data.dataloader import default_collate
 
-from flash.core.data.input_transform import InputTransform, InputTransformPlacement
-from flash.core.registry import FlashRegistry
+from flash.core.data.input_transform import Compose, InputTransform, LambdaInputTransform
+from flash.core.data.transforms import ApplyToKeys
 from flash.core.utilities.stages import RunningStage
 
 
 def test_input_transform():
-
-    transform = InputTransform(running_stage=RunningStage.TRAINING)
-
-    assert "InputTransform(running_stage=train, transform={<InputTransformPlacement.COLLATE: 'collate'>" in str(
-        transform
-    )
-
     def fn(x):
         return x + 1
 
-    transform = InputTransform.from_train_transform(transform=fn)
-    assert transform.transform == {InputTransformPlacement.PER_SAMPLE_TRANSFORM: fn}
+    class MyTransform(InputTransform):
+        def per_batch_transform(self) -> Callable:
+            return super().per_batch_transform()
```

Comment:
Unnecessary?

or is it to signal that it's overridden?

if so, i'd do

```suggestion
        def per_batch_transform(self) -> Callable:
            ...
```

same for the rest in this file

Reply:
Yes, done on purpose, so the function gets filtered.